data/mods/More_Survival_Tools/item_groups.json

```diff
@@ -0,0 +1,226 @@
+      ["bottle_metal", 2]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "subway",
+    "items":[
+      ["tinder", 1],
+      ["hobo_stove", 1]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "trash_forest",
+    "items":[
+      ["tinder", 1]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "museum_primitive",
+    "items":[
+      ["spear_stone", 50],
+      ["atlatl", 50],
+      ["javelin", 30],
+      ["hand_axe", 50],
+      ["throwing_stick", 50],
+      ["copper_knife", 10],
+      ["copper_ax", 10]
+    ]
+  },
+  {
+    "type" : "item_group",
+    "id" : "livingroom",
+    "items":[
+      ["gloves_work", 2]
+    ]
+  },
+  {
+    "type" : "item_group",
+    "id" : "home_hw",
+    "items":[
+      ["gloves_work", 45]
+    ]
+  },
+  {
+    "type" : "item_group",
+    "id" : "dresser",
+    "items":[
+      ["boxer_briefs", 45]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "male_underwear_bottom",
+    "items":[
+      ["boxer_briefs", 50]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "allclothes",
+    "items":[
+      ["boxer_briefs", 20],
+      ["gloves_work", 5]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "survivorzed_extra",
+    "items":[
+      ["hat_hooded", 1]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "survivorzed_gloves",
+    "items":[
+      ["gloves_work", 24]
+    ]
+  },
+  {
+    "type" : "item_group",
+    "id" : "construction_worker",
+    "items":[
+      ["gloves_work", 45]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "produce",
+    "items":[
+      ["garlic", 5]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "hydro",
+    "items":[
+      ["garlic", 3]
+    ]
+  },{
+    "type" : "item_group",
+    "id" : "stash_food",
+    "items":[
+      ["cheese_hard", 15]
+    ]
+  },{
+"type" : "item_group",
+"id" : "forage_spring",
+"subtype": "distribution",
+"entries" : [
+      { "item": "egg_bird", "prob": 15, "count-min": 2, "count-max": 5 },
+      { "item": "egg_reptile", "prob": 5, "count-min": 2, "count-max": 5 },
+      { "item": "rhubarb", "prob": 5, "count-min": 2, "count-max": 5 },
+      { "item": "wild_herbs", "prob": 10 },
+      { "item": "garlic", "prob": 5, "count-min": 1, "count-max": 6 },
+      { "item": "buckwheat", "prob": 5, "count-min": 1, "count-max": 2 },
+      { "item": "withered", "prob": 5 }
+    ]
+  },{
+"type" : "item_group",
+"id" : "forage_summer",
+"subtype": "distribution",
+"entries" : [
+      { "item": "veggy_wild", "prob": 100, "count-min": 1, "count-max": 2 },
+      { "group": "forage_mushroom", "prob": 50 },
+      { "item": "egg_bird", "prob": 15, "count-min": 2, "count-max": 5 },
+      { "item": "egg_reptile", "prob": 5, "count-min": 2, "count-max": 5 },
+      { "item": "wild_herbs", "prob": 10 },
+      { "item": "garlic", "prob": 10, "count-min": 1, "count-max": 6 },
+      { "item": "buckwheat", "prob": 10, "count-min": 1, "count-max": 4 },
+      { "item": "withered", "prob": 10 }
```

Comment:
Addition of withered plants to forage itemgroups can be yanked out at your discretion, but keeping it as mod-only works too. Only important thing here in the forage entries is adding garlic.